08_project/alextmz/cmd/puppy-server/main.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/anz-bank/go-course/08_project/alextmz/pkg/puppy"
+	"github.com/anz-bank/go-course/08_project/alextmz/pkg/puppy/store"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	s1 := store.NewMapStore()
+	s1p1 := puppy.Puppy{Breed: "Dogo", Colour: "White", Value: 500}
+	fmt.Printf("Creating puppy on Mapstore : ")
+	fmt.Printf("%#v (ID should start at 0 and be populated after creation)\n", s1p1)
+	_ = s1.CreatePuppy(&s1p1)
+
+	fmt.Fprintf(out, "Created puppy              : %#v\n", s1p1)
```

Comment:
IMO iut should be something like: `fmt.Fprintf("%-30s : %#v\n", "Created puppy", s1p1)` - and construct it the same way in tests

Reply:
done